PetRescue/src/components/TextInput.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {View} from 'react-native';
+
+const TextInput = () => {
+  return <View></View>;
```

Comment:
Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```